src/components/navigation-bar/navigation-bar.style.js

```diff
@@ -1,13 +1,11 @@
 import styled, { css } from "styled-components";
-import { space } from "styled-system";
+import { padding, flexbox } from "styled-system";
 import { baseTheme } from "../../style/themes";
-import Box from "../box";
 
-const StyledNavigationBar = styled(Box).attrs({ as: "nav" })`
-  padding: 0 40px;
+const StyledNavigationBar = styled.div.attrs({ as: "nav" })`
```

Comment:
I think this could just be

```suggestion
const StyledNavigationBar = styled.nav`
```